packages/legend-query/src/stores/QueryBuilderPropertyEditorState.ts

```diff
@@ -205,6 +203,10 @@ export class QueryBuilderDerivedPropertyExpressionState {
   ) {
     this.path = getPropertyPath(propertyExpression);
     this.title = getPropertyChainName(propertyExpression, true);
+    observe_AbstractPropertyExpression(
```

Comment:
Same comment, ideally we should call this observable before new QueryBuilderDerivedPropertyExpressionState...